src/Packing.cpp

```diff
@@ -0,0 +1,67 @@
+//
+// Created by rolf on 03-03-21.
+//
+
+#include "Packing.h"
+#include "utilities.h"
+#include <cmath>
+
+RobotCommandPayload createLowLevelCommand(const proto::RobotCommand& proto,  const proto::World& world, bool isYellow){
```

Comment:
Is it possible to make `world` optional? It should be possible to drive robots manually from the AI interface without a world instance running.

Reply:
It is, but there's no point, since we can also construct an empty world, and then conclude we cannot find the robot in this call.. This is what we do now. I personally like this a bit better, since it is less complex.

Reply:
Is it easier to create an empty world, or wrap it in an std::optional?

Reply:
The first; that's what's already is done :)